src/main/resources/public/alto-boot/static/js/main.js

```diff
@@ -264,17 +268,19 @@ function load_input(username, study_condition)
 				window.alert("error");
 				return;
 			}
-			topicsnum = json.topicsnum;
-			corpusname = json.corpusname;
-			render_input(json, study_condition);
-			fillDocToIndexMap();
-			$('#loading').modal('hide');
-			startSeconds = new Date().getTime() / 1000;
-			addStartLogs(startSeconds, study_condition);
-			count_down();
-			if(mainWindow.global_study_condition == TA_CONDITION || mainWindow.global_study_condition == TR_CONDITION){
-				followScroll();
-			}
+			setTimeout(() => {
+                topicsnum = json.topicsnum;
+                corpusname = json.corpusname;
+                render_input(json, study_condition);
+                fillDocToIndexMap();
+                $('#loading').modal('hide');
+                startSeconds = new Date().getTime() / 1000;
+                addStartLogs(startSeconds, study_condition);
+                count_down();
+                if(mainWindow.global_study_condition == TA_CONDITION || mainWindow.global_study_condition == TR_CONDITION){
+                    followScroll();
+                }
+            }, 1000);
```

Comment:
even with a 1 second time out, it's still a potential race condition although i haven't encountered it since. it's on my todo to remove the race condition altogether